src/utils/api/api.js

```diff
@@ -0,0 +1,19 @@
+async function api(endpoint, init = {}) {
+  let url;
+  init.credentials = 'include';
+  switch (endpoint) {
+    case 'menu':
+      url = '/admin-api/menu?_format=json';
+      break;
+    default:
+      break;
+  }
+
+  const data = await fetch(
```

Comment:
Is there a reason for this mix of await and promises?

Reply:
@dawehner how else would you write this?

Reply:
Couldn't you use ```const res = await fetch(
  process.env.REACT_APP_DRUPAL_BASE_URL + url,
  init,
);
const data = await res.json();
return data;
```